### DISCUSSION TITLE
Uncaught (in promise) ReferenceError: business is not defined

### DISCUSSION THREAD

**Discussion by ohsangbong** [kimytsc/covid-rest-vaccine-macro#29] [Q&A] [answered]
콘솔 실행시키면 저 에러가 계속 찍혀나오는데 상관없는것인가요?? 제대로 동작하고 있는건지 궁금합니다

**Comment by moguv587** [comment, marked as answer]
line 154
business.vaccineQuantity.list.forEach(item => {
change to 
bussiness.vaccineQuantity.list.forEach(item => {

missed a 's'

**Reply by ohsangbong** [reply]
Thank you!

**Comment by kimytsc** [comment]
죄송합니다.. 네이버 백신 필터 프로세스를 수정하는 과정에서 오타가 나서 오류가 발생했었습니다.

조금전 다시 수정하여 올렸으니 이제 정상적으로 사용하실 수 있습니다.
